components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/ProxyFacetSupport.java

```diff
@@ -212,7 +212,7 @@ protected Content fetch(String url, Context context, Content stale) throws IOExc
 
       Payload payload = new HttpEntityPayload(response, entity);
       final Content result = new Content(payload);
-      result.getAttributes().set(Content.CONTENT_LAST_MODIFIED, extractLastModified(response.getLastHeader(HttpHeaders.LAST_MODIFIED)));
+      result.getAttributes().set(Content.CONTENT_LAST_MODIFIED, extractLastModified(request.getURI(), response.getLastHeader(HttpHeaders.LAST_MODIFIED)));
```

Comment:
Ack.

Well, the information is useful, but if remote server is broken/buggy/misconfigured, this would probably result in log flood (as all the artifacts from remote would have bad header). OTOH, if this would be DEBUG, NX admin would not notice this problem in the first place...